Lib/ufo2ft/postProcessor.py

```diff
@@ -155,7 +155,7 @@ def process_glyph_names(self, useProductionNames=None):
 
             if useProductionNames:
                 logger.info("Renaming glyphs to final production names")
-                self._rename_glyphs_from_ufo()
+                self.rename_glyphs(self.otf, self._build_production_names())
```

Comment:
I think we want to keep this line as it was before

self._rename_glyphs_from_ufo()

Reply:
Yep that was my arse clowing. Give me a sec.